packages/fxa-settings/src/components/App/RememberPassword/index.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import 'mutationobserver-shim';
```

Comment:
What's this do?

Reply:
Good question, do you know? I followed the test format for other Settings components, and this import is in a lot of Settings test files. I tested commenting it out from the other files and saw no errors, so I couldn't tell if it was doing something under the hood.

Reply:
I've seen it before, but had to search for what it does. [This issue](https://github.com/react-hook-form/react-hook-form/issues/312) makes me think it's a react-hook-form thing that we might need to listen to form updates? If it works without, I'd say leave it out.